apps/nameguard.io/src/app/docs/displayed-name/page.tsx

```diff
@@ -138,19 +159,37 @@ export default function DisplayedNamePage() {
             <div className="w-full flex justify-center items-center font-mono">
               <pre>
                 {
-                  "<DisplayedName name\n textStylingClasses='text-purple-400' />"
+                  "<DisplayedName name\n textStylingClasses='\nens-webfont font-black' />"
                 }
               </pre>
             </div>
 
             <div className="flex items-center justify-center">
               <DisplayedName
-                textStylingClasses="text-purple-400"
+                textStylingClasses="ens-webfont font-black"
```

Comment:
Why do so many of these have "font-black" in their text styling? This comment applies across the whole page not just this line of code.

Reply:
@FrancoAguzzi What I really want to see on the demo page is that:

1. DisplayedName shows the tooltip in ens-webfont
2. Some of the DisplayedName shows custom font styling for the TruncatedText (but not the tooltip, outside of request 1 above)

Reply:
It is an example of how users can customize tooltip content and / or text content styles

Reply:
@lightwalker-eth `TruncatedText` examples were updated to use plain strings